src/main/java/net/sf/jabref/gui/importer/fetcher/GoogleScholarFetcher.java

```diff
@@ -193,13 +165,21 @@ private static void runConfig() throws IOException {
     }
 
     private String getCitationsFromUrl(String urlQuery, Map<String, JLabel> ids) throws IOException {
-        String cont = new URLDownload(urlQuery).downloadToString(StandardCharsets.UTF_8);
-        Matcher m = GoogleScholarFetcher.BIBTEX_LINK_PATTERN.matcher(cont);
+        String cont = URLDownload.createURLDownloadWithBrowserUserAgent(urlQuery).downloadToString(StandardCharsets.UTF_8);
+
+        Matcher m = GoogleScholarFetcher.SCHOLAR_ID_PATTERN.matcher(cont);
         int lastRegionStart = 0;
 
         while (m.find()) {
-            String link = m.group(1).replace("&amp;", "&");
-            link = link+"&oe=utf-8"; // append param 'oe=utf-8' to tell google to serve UTF-8 encoded results
+
+            String citationsPageURL = CITATIONS_PAGE_URL_BASE+m.group(1)+CITATIONS_PAGE_URL_SUFFIX;
+
+            String citationsPage = URLDownload.createURLDownloadWithBrowserUserAgent(citationsPageURL).downloadToString(StandardCharsets.UTF_8);
+
+            Matcher citationPageMatcher = GoogleScholarFetcher.BIBTEX_LINK_PATTERN.matcher(citationsPage);
```

Comment:
That's unfortunate. 
I did some research and I think I found something. 
Google stores the setting to display bib-links in a cookie. Sending the cookie `GSP=IN=d192d757fd09588a+7e6cc990821af64:CF=4` along the first query directly shows the Import into Bibtex link, 
````
<a href="https://scholar.googleusercontent.com/scholar.bib?q=info:b2pGeL14LLMJ:scholar.google.com/&amp;output=citation&amp;scisig=AAGBfm0AAAAAV-vafm6NWi20weGxxou9W2xi8GzZ8YCf&amp;scisf=4&amp;ct=citation&amp;cd=0&amp;hl=en" class="gs_nta gs_nph">Import into BibTeX</a>
````

Appearently, the part after GSP=IN=... is not that important since my real id had 63 at the end instead of 64.